lib/index.d.ts

```diff
@@ -290,6 +290,21 @@ declare namespace Realm.Sync {
         isAdmin: boolean;
     }
 
+    class Credentials {
+        static usernamePassword(username: string, password: string, createUser?: boolean): Credentials;
+        static facebook(token: string): Credentials;
+        static google(token: string): Credentials;
+        static anonymous(): Credentials;
+        static nickname(value: string, isAdmin?: boolean): Credentials;
+        static azureAD(token: string): Credentials;
+        static jwt(token: string): Credentials;
+        static adminToken(token: string): Credentials;
+
+        readonly provider: string;
+        readonly token: string;
```

Comment:
And this is called `userIdentifier`

Reply:
In .NET and Cocoa we use `token`, although as mentioned, Cocoa's properties are private. I'll speak with Kenneth and see which one he likes better - I don't have a strong opinion on the matter :)